tests/testthat/_snaps/po_create.md

```diff
@@ -0,0 +1,18 @@
+# the user is told what's happening
+
+    Code
+      po_create("jp", verbose = TRUE)
+    Message <simpleMessage>
+      Creating 'jp' R translation
+      Running msgint on './po/R-jp.po' succeeded:
+        Created ./po/R-jp.po.
+
+---
+
+    Code
+      po_create("jp", verbose = TRUE)
+    Message <simpleMessage>
+      Updating 'jp' R translation
+      Running msgmerge on './po/R-jp.po' succeeded:
```

Comment:
I copied this from #257, but it feels a bit wordy to me. What do you think about not showing the call by default?

Reply:
Hmm it's coming from `verbose=TRUE` (now the default), are you suggesting we flip the default, or that we implement `verbose=0,1,2,...`?

Reply:
I don't think it's necessary to routinely display a message for success. Adding levels of verbosity is unlikely to be worth the effort IMO, so I'd be in favour of `verbose = FALSE` by default and not passing in `verbose` from the wrapping function.

Reply:
Oh that's not quite right; I think on success we want to see the stdout/stderr from the command line tool, but not the call.

Reply:
I've tweaked the code to behave the way that I want just to give you a concrete proposal to look at. Let me know what you think.